### PR TITLE
[ios] Update project for Xcode 10.2/iOS 12.2 (release-liquid)

### PR DESCRIPTION
**Pull request by jmkiley**
Cherry-picks #14241 

cc @fabian-guerra

### CODE REVIEW

File: platform/ios/Integration Tests/MGLCameraTransitionTests.mm [hunk 1]
```diff
@@ -277,7 +277,7 @@ - (void)testFlyToCameraInDelegateMethod {
 
                 // Now set another coordinate.
                 MGLMapCamera *camera = [MGLMapCamera cameraLookingAtCenterCoordinate:target2
-                                                                        fromDistance:altitude
+                                                                            altitude:altitude
```
Comment (fabian-guerra) [inline platform/ios/Integration Tests/MGLCameraTransitionTests.mm]: Why did we removed this test?

Comment (julianrex) [reply platform/ios/Integration Tests/MGLCameraTransitionTests.mm]: One of our deprecated methods. We could add another, and silence the warning.

Comment (fabian-guerra) [reply platform/ios/Integration Tests/MGLCameraTransitionTests.mm]: I understand. Thank you for explaining.

**Comment by julianrex** [comment]
Closing since this was [merged elsewhere](https://github.com/mapbox/mapbox-gl-native/pull/14274)

**Pull request closed without merging**